addon/lib/field.js

```diff
@@ -405,9 +442,15 @@ export default Base.extend(Evented, {
    * @internal
    */
   _validateTextQuestion() {
-    return validate("length", this.get("answer.value"), {
-      max: this.get("question.textMaxLength") || Number.POSITIVE_INFINITY
-    });
+    return [
+      ...this.validator.validate(
+        this.get("answer.value"),
+        this.getWithDefault("question.meta.formatValidators", [])
+      ),
+      validate("length", this.get("answer.value"), {
+        max: this.get("question.textMaxLength") || Number.POSITIVE_INFINITY
+      })
+    ];
```

Comment:
You need to return a `Promise` here, the `validate` task does not support arrays as return value of a validation function

Reply:
Discussed: return a single value here

Reply:
Florian said arrays are fine.

We already have functions returning an array ([L527](https://github.com/projectcaluma/ember-caluma/blob/master/addon/lib/field.js#L527)) and the return value from the function is transformed to an array anyway ([L376](https://github.com/projectcaluma/ember-caluma/blob/master/addon/lib/field.js#L376)).